tests/gpu/linalg_test/Add_test.cpp

```diff
@@ -126,6 +126,33 @@ namespace AddTest {
     }
   }
 
+  // Test tensor-to-tensor addition with mixed types
+  TEST_P(AddTestAllShapes, gpu_tensor_add_tensor_mixed_types) {
```

Comment:
It seems that this test already covers `TEST_P(AddTestAllShapes, gpu_tensor_add_tensor_all_types)`. Is it fine the remove the following one?